.env

```diff
@@ -88,6 +88,8 @@ ESPRESSO_SEQUENCER_STAKE_TABLE_CAPACITY=10
 # Foundry
 # The mnemonic used by foundry to deploy contracts.
 MNEMONIC="test test test test test test test test test test test junk"
+# The mnemonic offset, starts at zero
+MNEMONIC_OFFSET=0
 # The RPC URL for deploying to the sepolia network.
 SEPOLIA_RPC_URL=
```

Comment:
nit: should change this as well to `RPC_URL`

Reply:
Currently, the `SEPOLIA_RPC_URL` is specific to Sepolia gas profiling tests. I think we can consider network-labeled RPC URLs as we expand tests to other networks. If managing these becomes unwieldy, we can then consolidate to `RPC_URL`. What do you think?

Reply:
okay that makes sense! yea lets keep it `SEPOLIA_RPC_URL` here for now